kratos/tests/test_model.py

```diff
@@ -53,6 +53,9 @@ def test_model(self):
     def _create_and_save_model(self,file_name, serializer_flag):
         current_model = KratosMultiphysics.Model()
 
+        # Setting the data value container
+        current_model.SetValue(KratosMultiphysics.TEMPERATURE, 293.0)

```

Comment:
this for example becomes

~~~py
    current_model.GetDataValueContainer().SetValue(KratosMultiphysics.TEMPERATURE, 293.0)
~~~